### PR TITLE
[docs] add documentation for 2FA

### PR DESCRIPTION
**Pull request by tcdavis**
# Why

2FA is ready for release.  Adding it to docs will make it easier for people to find and know how to use

# Test Plan

`yarn dev`

### CODE REVIEW

File: docs/pages/accounts/two-factor.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+- regenerate your recovery codes
+- disable two-factor authentication for your account
+
+You will need to provide a verification code before any changes to your 2FA settings will be applied.
+
+## Recovering Your Account
+
+### Recovery Codes
+
+While setting up 2FA, Expo provides you with recovery codes. In the event you lose your 2FA method, Expo will ask for these recovery codes to unlock 2FA. These codes are for one-time use and will no longer work after they are used to recover your account. To get new recovery codes, you'll need to reset your 2FA method.
```
Comment (wschurman) [inline docs/pages/accounts/two-factor.md:56]:
```suggestion
While setting up 2FA, Expo provides you with recovery codes. In the event you lose your 2FA method, a recovery code may be used in place of a one-time password. Each of these codes may only be used once. To get new recovery codes, you'll need to reset your 2FA method.
```

File: docs/pages/accounts/two-factor.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+### Secondary 2FA Methods
+
+By setting up multiple authentication methods associated with different physical devices, you can ensure you will not lose access to your account in the event you lose access to an individual device.
```
Comment (wschurman) [inline docs/pages/accounts/two-factor.md:60]:
```suggestion
By setting up multiple authentication methods associated with different physical devices, you can ensure you will not lose access to your account in the event a device is lost.
```

File: docs/pages/accounts/two-factor.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+## Changing Your Two-Factor Settings
+
+You can make changes to your two-factor settings from your [personal account settings](https://expo.io/settings).  You can:
+
+- add or remove authentication methods
+- set your default method
+- regenerate your recovery codes
+- disable two-factor authentication for your account
+
+You will need to provide a verification code before any changes to your 2FA settings will be applied.
```
Comment (wschurman) [inline docs/pages/accounts/two-factor.md:50]:
```suggestion
You will need to provide a one-time password to make any changes to your 2FA settings.
```

File: docs/pages/accounts/two-factor.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+### SMS Messages
+
+Provide a mobile phone number to receive a short-lived token from Expo. Once you enter the one-time password, your mobile phone number will be used for 2FA when logging into your account.  Codes received via SMS will be valid for at least 10 minutes, so you may receive the same code if you request a token more than once within this window.
+
+> **Note:** If you set an SMS device as your default 2FA method, you will be sent a verification code automatically whenever you take an action that requires a 2FA code.
+
+### Recovery Codes
+
+As part of setting up your account to use two-factor authentication you will receive a list of recovery codes.  These codes can be entered anywhere you need to provide a one-time password if you lose access to the device(s) that have your authenticator app or that you use to receive SMS messages.  Once you use a recovery code, you will not be able to use it again.
```
Comment (wschurman) [inline docs/pages/accounts/two-factor.md:35]:
```suggestion
As part of setting up two-factor authentication on your account you will receive a list of recovery codes.  These codes may be used in lieu of a one-time password when you lose access to your authenticator device(s).  Each recovery code may only be used once.
```

File: docs/pages/accounts/two-factor.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+- Authy
+- 1Password
+- Google Authenticator
+- Microsoft Authenticator
+
+Expo will provide a QR code for scanning with your authenticator app. Open your app of choice, and scan the QR Code. The app will provide a confirmation code to enter on Expo. Enter the code to finish activating 2FA via your authenticator app. 
+
+### SMS Messages
+
+Provide a mobile phone number to receive a short-lived token from Expo. Once you enter the one-time password, your mobile phone number will be used for 2FA when logging into your account.  Codes received via SMS will be valid for at least 10 minutes, so you may receive the same code if you request a token more than once within this window.
```
Comment (wschurman) [inline docs/pages/accounts/two-factor.md:29]:
```suggestion
Provide your mobile phone number to receive Expo one-time passwords via SMS.  One-time passwords received via SMS will be valid for at least 10 minutes, so you may receive the same code multiple times within this window.
```

File: docs/pages/accounts/two-factor.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+
+### Authenticator Apps
+
+Expo accepts any authenticator app that supports Time-based One-time Passwords (TOTP), such as the following:
+
+- Last Pass Authenticator
+- Authy
+- 1Password
+- Google Authenticator
+- Microsoft Authenticator
```
Comment (wschurman) [inline docs/pages/accounts/two-factor.md:23]:
```suggestion
- [Last Pass Authenticator](https://lastpass.com/auth/)
- [Authy](https://authy.com/)
- [1Password](https://support.1password.com/one-time-passwords/)
- [Google Authenticator](https://support.google.com/accounts/answer/1066447)
- [Microsoft Authenticator](https://www.microsoft.com/en-us/account/authenticator)
```

File: docs/pages/accounts/two-factor.md [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+
+Expo will provide a QR code for scanning with your authenticator app. Open your app of choice, and scan the QR Code. The app will provide a confirmation code to enter on Expo. Enter the code to finish activating 2FA via your authenticator app.
```
Comment (wschurman) [inline docs/pages/accounts/two-factor.md:25]:
```suggestion
Expo will provide a QR code to scan with your authenticator app during setup.  The app will provide a confirmation code to enter on Expo.  Enter the code to finish activating 2FA via your authenticator app. 
```

File: docs/pages/accounts/two-factor.md [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+
+You can enable two-factor authentication from your [personal account settings](https://expo.io/settings).
+
+## Two-Factor Authentication Methods
+
+You can receive 2FA codes through an authenticator app or via SMS message
+
+### Authenticator Apps
+
+Expo accepts any authenticator app that supports Time-based One-time Passwords (TOTP), such as the following:
```
Comment (wschurman) [inline docs/pages/accounts/two-factor.md:17]:
```suggestion
Expo accepts any authenticator app that supports Time-based One-time Passwords (TOTP) including:
```

File: docs/pages/accounts/two-factor.md [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+
+Two factor authentication provides an extra layer of security when logging in to expo.io, the Expo Go app, and command line tools.  With 2FA enabled, you will need to provide a secure code in addition to your username and password to access your account.
+
+## Enabling Two-Factor Authentication (2FA)
+
+You can enable two-factor authentication from your [personal account settings](https://expo.io/settings).
+
+## Two-Factor Authentication Methods
+
+You can receive 2FA codes through an authenticator app or via SMS message
```
Comment (wschurman) [inline docs/pages/accounts/two-factor.md:13]:
```suggestion
You can receive 2FA codes through an authenticator app or via SMS message.
```

File: docs/pages/accounts/two-factor.md [hunk 10]
```diff
@@ -0,0 +1,64 @@
+---
+title: Two-Factor Authentication
+---
+
+Two factor authentication provides an extra layer of security when logging in to expo.io, the Expo Go app, and command line tools.  With 2FA enabled, you will need to provide a secure code in addition to your username and password to access your account.
```
Comment (wschurman) [inline docs/pages/accounts/two-factor.md:5]:
```suggestion
Two-factor authentication provides an extra layer of security when logging in to expo.io, the Expo Go app, and command line tools.  With two-factor authentication enabled, you will need to provide a secure short-lived code in addition to your username and password to access your account.
```
(we haven't defined the abbreviation 2FA yet as of this usage)

**Review by wschurman — APPROVED** [review]
All comments are suggestions but are not blocking. Make sure to fix lint before landing though.

File: docs/pages/accounts/two-factor.md (hunk 2 above)
Comment (ide) [reply docs/pages/accounts/two-factor.md]: maybe "reset or lost"? just to get people thinking about resetting their phones

**Pull request merged**